.github/workflows/feg-workflow.yml

```diff
@@ -88,7 +88,7 @@ jobs:
         if: always()
         id: feg-precommit
         run: |
-          go get gotest.tools/gotestsum
+          go install gotest.tools/gotestsum@latest
```

Comment:
Nit: I generally shy away from pinning to latest, good way to get spontaneous CI breakages

Reply:
Make sense, is pinned  to `1.8.0` now.